d2core/d2input/d2input_ebiten/backend_ebiten.go

```diff
@@ -0,0 +1,65 @@
+package d2input_ebiten
+
+import (
+	"github.com/OpenDiablo2/OpenDiablo2/d2core/d2input/keyboard"
+	"github.com/OpenDiablo2/OpenDiablo2/d2core/d2input/mouse"
+
+	"github.com/hajimehoshi/ebiten"
+	"github.com/hajimehoshi/ebiten/inpututil"
+)
+
+func keyToEbiten(k keyboard.Key) ebiten.Key {
+	return ebiten.Key(k)
```

Comment:
Does this just not assume that all of the constants are the same between ebiten and the values that you added? How will this work when you try to add another input backend?

Reply:
A map is now explicitly defined, and not exported. It's now part of init for the input backend.